The original file is provided by Busybox, and 
can be found in the Buildroot package directory. 

This adds devices found on arm64 hardware and 
rules for fuse and crda as well as automount 
options

Signed-off-by: Andreas Ziegler <br015@umbiko.net>

--- a/etc/mdev.conf
+++ b/etc/mdev.conf
@@ -17,10 +17,12 @@
 
 # Typical devices
 tty		root:tty 666
-tty[0-9]*	root:tty 660
-vcsa*[0-9]*	root:tty 660
-ttyS[0-9]*	root:root 660
-ttyUSB[0-9]*	root:root 660
+tty[0-9]	root:tty 660
+tty[1-6][0-9]	root:tty 660 @rm $MDEV
+vcs[au]*[0-9]*	root:tty 660
+ttyS[0-9]*	root:tty 660
+ttyUSB[0-9]*	root:tty 660
+ttyAMA[0-9]*	root:tty 660
 
 # alsa sound devices
 snd/pcm.*	root:audio 660
@@ -35,5 +37,11 @@
 input/mouse[0-9]	root:root 640
 input/ts[0-9]		root:root 600
 
+fuse	root:root 666
+
 # load modules
 $MODALIAS=.*	root:root 660 @modprobe "$MODALIAS"
+
+# auto mount usb devices
+sd[a-z][0-9]*	root:disk 660 */var/lib/mdev/automount
+mmcblk[0-9]p[0-9]	root:disk 660 */var/lib/mdev/automount
